README.rst

```diff
@@ -163,35 +163,67 @@ with:
 
 	$ grunt watch
 
-It is also possible to develop figure in docker without installing anything locally.
-The Docker image is built on top of ``openmicroscopy/omero-web-standalone:latest``, so you will have a fully functional
-omero-web environment while developing ``omero-figure``. 
-First build the Docker image:
+
+Local install
+*************
+
+You need an `omero-web` environment. You can either install it locally or use Docker (see below).
+To install locally, we recommend that you use `conda` to install `omero-py` as described 
+at https://github.com/ome/omero-py and then perform a developer install of `omero-figure`
+which will include `omero-web`:
 
 ::
 
-   $ docker build -t figure-devel .
+    $ cd omero-figure
+    $ pip install -e .
+
+You'll also need to run the `omero config` steps detailed above.
+See instructions at https://omero.readthedocs.io/en/latest/developers/Web/Deployment.html
+for running the development server.
+
+Using Docker
+************
 
+It is also possible to develop figure in docker without creating a python environment locally.
+The Docker image at ``omero-figure/Dockerfile`` is built on top of
+`openmicroscopy/omero-web-standalone:latest <https://hub.docker.com/r/openmicroscopy/omero-web-standalone>`_,
+so you will have a fully functional omero-web environment while developing ``omero-figure``.
+This Docker image includes OMERO.figure. You can either replace this with a link to your local ``omero-figure`` repo
+or you can use the Docker copy of OMERO.figure for development.
+
+In either case, start by building the Docker image and specify the server you wish to connect to:
 
-To develop ``omero-figure`` you can either make all the changes within the Docker container itself by running:
 ::
 
+   $ cd omero-figure
+   $ docker build -t figure-devel .
+   $ export OMEROHOST=demo.openmicroscopy.org
+
+To use your local ``omero-figure`` repo for development, you can mount this in place of the Docker ``omero-figure``.
+You'll need `npm` and `grunt` installed locally and run `grunt watch` during development as described above.
+You can refresh the Docker hosted page at http://localhost:4080/figure to see changes:
 
-    $ docker run -ti -e OMEROHOST=YOUR_HOST -p 4080:4080 figure-devel
+::
+
+    $ cd /PATH_TO_GIT_REPO/omero-figure
+    $ grunt build
+    $ grunt watch   # to see updates
```

Comment:
This needs a separate terminal. 
Grunt watch is blocking the terminal, so the ``docker run ...`` cmd below must be run in a separate one.

Reply:
Done in 2659ac00